packages/paste-core/components/button/src/Button.tsx

```diff
@@ -280,15 +328,6 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
   },
 );
 
-Button.defaultProps = {
-  as: "button",
-  fullWidth: false,
-  disabled: false,
-  loading: false,
-  type: "button",
-  variant: "primary",
-};
```

Comment:
This was deprecated and the LSP was screaming about it. I shifted this up to be destructured in the props of the base Button component.